spec/shared_examples.rb

```diff
@@ -29,6 +29,19 @@
     end
   end
 
+  context 'Prepare' do
+    it 'should prepare resource' do
+      res = @manager.prepare(@resource['id'])
+      expect(res.instance_variable_get(:@entity)).to be_nil
+      expect(res.instance_variable_get(:@entity_id)).to eq(@resource['id'])
+
+      expect(res.entity).to be_truthy
+      expect(res.entity).to include(base_attr => @name)
+      expect(res.instance_variable_get(:@entity)).to be_truthy
```

Comment:
What is the real purpose of testing this? That it is cached? What about calling entity twice and expecting just one request? Testing instance variable is really an anti pattern.